misk/src/main/kotlin/misk/client/HttpClientsConfigBackwardsCompatibility.kt

```diff
@@ -0,0 +1,146 @@
+package misk.client
+
+import com.fasterxml.jackson.annotation.JsonAlias
+import com.fasterxml.jackson.databind.type.TypeFactory
+import java.time.Duration
+
+data class BackwardsCompatibleEndpointConfig(
+    //Common fields
```

Comment:
nit: spacing and missing ` ` after `//`

Reply:
Fixed the space after `//`, but won't fix the spacing before as that's how our IntelliJ rules reformat it.